docusaurus/README.md

```diff
@@ -18,26 +18,63 @@ yarn install
 yarn build
 ```
 
-At this point you will see any broken links that docusaurus was able to find.
+`yarn build` will build Docusaurus site in `docusaurus/build` directory.
 
 ## Developing Locally
 
+If you want to make changes to the documentation, you can run docusaurus locally in a way that would
+listen to any source docs changes and live-reload them:
+
 ```bash
 yarn start # any changes will automatically be reflected in your browser!
 ```
 
-## Making Changes
+All the content for docs.airbyte.com lives in the `/docs` directory in this repo. All files are
+markdown. Make changes or add new files, and you should see them in your browser!
```

Comment:
Caveat: adding new files and seeing them requires changing the sidebar (thanks for separating out the next section to make it clearer!), so you can't see this with yarn start. I believe if you need to change the sidebar, you have to run `build` again (unconfirmed)

Reply:
Mmmm, I'll double check!